pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -884,6 +885,12 @@ in
     '';
   };
 
+  dailies-nvim = super.dailies-nvim.overrideAttrs {
+    dependencies = with super; [
+      dailies
+    ];
```

```suggestion
    runtimeDeps = [
      dailies
    ];
```

`dependencies` is for other vimPlugin dependencies

---

Oh my bad, updated!